cumulus/primitives/utility/src/lib.rs

```diff
@@ -763,7 +762,7 @@ mod test_trader {
 /// Implementation of `xcm_builder::EnsureDelivery` which helps to ensure delivery to the
 /// parent relay chain. Deposits existential deposit for origin (if needed).
 /// Deposits estimated fee to the origin account (if needed).
-/// Allows to trigger additional logic for specific `ParaId` (e.g. open HRMP channel) (if neeeded).
+/// Allows to trigger additional logic for specific `ParaId` (e.g. open HRMP channel) (if needed).
```

Comment:
```suggestion
/// Allows triggering of additional logic for a specific `ParaId` (e.g. to open an HRMP channel) if needed.
```

Reply:
Yeah, there's quite a lot of grammatical errors as well, though I mostly focused on spelling mistakes in this PR, maybe I'll make a follow up PR for this (though it's harder to detect grammatical errors with tools).

Reply:
Fixed